Networking/Repository/AbstractRepository.swift

```diff
@@ -133,18 +133,25 @@ open class AbstractRepository {
     fileprivate let _sessionManager: SessionManagerType
     fileprivate let _requestExecutor: RequestExecutorType
     
+    fileprivate let _defaultHeaders: [String: String]?
+    
     public init(networkingConfiguration: NetworkingConfiguration,
                 requestExecutor: RequestExecutorType,
-                sessionManager: SessionManagerType) {
+                sessionManager: SessionManagerType,
+                defaultHeaders: [String: String]? = .none) {
         _networkingConfiguration = networkingConfiguration
         _requestExecutor = requestExecutor
         _sessionManager = sessionManager
+        _defaultHeaders = defaultHeaders
     }
     
-    public init(networkingConfiguration: NetworkingConfiguration, sessionManager: SessionManagerType) {
+    public init(networkingConfiguration: NetworkingConfiguration,
+                sessionManager: SessionManagerType,
+                defaultHeaders: [String: String]? = .none) {
         _networkingConfiguration = networkingConfiguration
```

Comment:
Shouldn't this init call the upper one with `requestExecutor ` argument in `defaultRequestExecutor(networkingConfiguration: networkingConfiguration)` ?